gapis/replay/batch.go

```diff
@@ -100,6 +100,139 @@ func (m *manager) batch(ctx context.Context, e []scheduler.Executable, b schedul
 	}
 }
 
+var initialID string
```

Comment:
Before I merge this I will move this into the database. Right now this will completely break with > 1 replay type.